mkdocs.yml

```diff
@@ -1,5 +1,5 @@
 site_name: SimpleLogin Docs
-site_url: https://simplelogin.io/docs/
+site_url: https://simplelogin.io/
```

Comment:
I think we need to revert the change in this file :)

Reply:
> I think we need to revert the change in this file :)

Yeah, makes sense after you told me what it does.